mapie/control_risk/crc_rcps.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Optional, Tuple
+
+sigma_init = .25
```

Comment:
QUESTION: What is this variable? Could it be defined by the user?

Reply:
I understand that this is the default value for a reference paper (following a copy and paste operation). So there's no problem.

Reply:
@SZiane, I think this variable should stay in the main class and should be put as an argument of the function (as it is not in the class anymore). This should stay as an attribute because only highly advanced user would like to change it. So in order to not add to many arguments to the class, I think this variable should stay hidden (but modifiable)

Reply:
See commit "UPD: change sigma_init". Tell me if it answer the problem

Reply:
Your change is good. I just add a comment in other part of the file.